website/docs/docs/cloud/about-cloud/dark-mode.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "dbt Cloud dark mode"
+id: dark-mode
+description: "Learn about theme switching in dbt Cloud"
+sidebar_label: dbt Cloud dark mode
+image: /img/docs/dbt-cloud/using-dbt-cloud/dark-mode.png
+---
+
+dbt Cloud supports **Light mode** (default), **Dark mode**, and **System mode** (automatic theme switching based on your browser’s preferences set) under the **Theme** section of your user profile. You can seamlessly switch between these modes directly from the profile menu, customizing your viewing experience.
+
+Your selected theme is stored in your user profile, ensuring a consistent experience across dbt Cloud.
+
+Theme selection applies across all areas of dbt Cloud, including the [IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud), [dbt Explorer](/docs/collaborate/explore-projects), [environments](/docs/environments-in-dbt), [jobs](/docs/deploy/jobs), and more. Learn more about customizing themes in [Enable dark mode in dbt Cloud](/docs/cloud/about-cloud/dark-mode#enable-dark-mode-in-dbt-cloud).
+
+## Prerequisites
+
+- You have a dbt Cloud account. If you don’t, try [dbt Cloud for free!](https://www.getdbt.com/signup)
+- Dark mode is currently available on the Developer plan. It will be available for all [plans](https://www.getdbt.com/pricing) in the future. We’ll be rolling it out gradually, so stay tuned for updates.
+
+## Enable dark mode in dbt Cloud
```

Comment:
Perhaps this entire section should just be labeled something like "Change themes in dbt Cloud". The page is already titled dark mode so I think it's ok to not reference dark mode directly.

Then both the enabled and disabled paragraphs really are combined, to enable dark mode, the user can just select it, and disabling is the opposite process. It feels strange to callout the IDE mechanism in the paragraph below "Disable dark mode".

Additionally, we could add a sentence explaining that "System" will respect browser theme for light or dark.